sei-db/sc/memiavl/import.go

```diff
@@ -12,8 +12,13 @@ import (
 )
 
 var (
-	nodeChanSize = 10000
-	bufIOSize    = 64 * 1024 * 1024
+	// Pipeline channel size - controls how many operations can be queued
+	// Profiling: peak 440k nodes, 1GB overhead. Small buffer causes frequent blocking.
+	nodeChanSize = 500000
+
+	// bufio.Writer buffer size - 128MB balances performance and memory usage
+	// Large buffer reduces syscalls and improves throughput for sequential writes
+	bufIOSize = 128 * 1024 * 1024
```

Comment:
Not sure if you know the `<<` trick for writing these: `128 << 20`.10 gives you KiB, 20 gives you MiB, 30 gives you GiB and so on.